api/src/main/java/api/TireFacade.java

```diff
@@ -0,0 +1,53 @@
+package api;
+
+import cz.muni.fi.pa165.pneuservis.backend.entity.TireManufacturer;
+import cz.muni.fi.pa165.pneuservis.backend.entity.TireProperties;
+import dto.TireDTO;
+import dto.TireManufacturerDTO;
+import dto.TirePropertiesDTO;
+
+import java.util.List;
+import java.util.Set;
+
+/***
+ * @author Jakub Palenik, 422453@mail.muni.cz
+ */
+
+public interface TireFacade {
+
+    /**
+     * Self explanatory
```

Comment:
Please write proper documentation, few words should be enough for self-explanatory functions.